utils/doclint/generateDotnetApi.js

```diff
@@ -280,7 +280,7 @@ function renderMember(member, parent, out) {
         throw new Error(`No Event Type for ${name} in ${parent.name}`);
       if (member.spec)
         output(XmlDoc.renderXmlDoc(member.spec, maxDocumentationColumnWidth));
-      if (parent && (classNameMap.get(parent.name) === type))
+      if (parent && (classNameMap.get(parent.name) === type) && !['Popup'].includes(name))
```

Comment:
We managed to not hardcode things like this in Python, any way to work around it?